python/src/llvm.cc

```diff
@@ -302,6 +303,22 @@ void init_triton_llvm(py::module &&m) {
         PassBuilder pb(nullptr /*targetMachine*/, tuningOptions, std::nullopt,
                        instrCbPtr);
 
+        std::string pluginFile =
+            mlir::triton::tools::getStrEnv("LLVM_PASS_PLUGIN");
```

Comment:
Can we add a `_PATH` suffix to be consistent with other (e.g., `TRITON_REPRODUCER_PATH`, `TRITON_LIBHIP_PATH`) environment variables? Aside from consistency, this makes the variable self-documenting in the sense it's pretty clear that we should specify a path rather than a pass plugin name or whatever.